### PR TITLE
Add "Slowest" gamespeed with Timestep of 80.

### PR DESCRIPTION
**Pull request by PeterAntal**
Some people just want to see the world burn. Really slowly.

Applies "slowest" game speed in the 4 stock mods.

As requested in #14784

### CODE REVIEW

File: mods/cnc/mod.yaml [hunk 1]
```diff
@@ -209,6 +209,10 @@ SpriteSequenceFormat: TilesetSpecificSpriteSequence
 ModelSequenceFormat: PlaceholderModelSequence
 
 GameSpeeds:
+	slowest:
+		Name: Slowest
+		Timestep: 80
+		OrderLatency: 3
```
Comment (GraionDilach) [inline mods/cnc/mod.yaml]: I would make this value 2 here, it'd still be longer than Slower's time window (160 ms vs 150 ms).

Comment (reaperrr) [reply mods/cnc/mod.yaml]: I agree.

Comment (PeterAntal) [reply mods/cnc/mod.yaml]: @reaperrr  & @GraionDilach  - Thx for feedback. Updated latency is applied here.

**Review by abcdefg30 — APPROVED** [review]

**Review by GraionDilach — APPROVED** [review]
:+1:

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/03a18a6f618d82f72acb787270a30df10cd37dca...3ef4e594264b231ee409682ba7b3c1b5e7482ac5)

**Pull request merged**